src/app/components/sandbox/ModeIcons.js

```diff
@@ -131,22 +131,28 @@ const getCurrentMode = ({
   setPreviewView,
 }: Props) => {
   const both = (
-    <ViewIcon onClick={setMixedView} active={showEditor && showPreview}>
-      <EditorIcon half />
-      <PreviewIcon half />
-    </ViewIcon>
+    <Tooltip title="Show both">
```

Comment:
Maybe you can make these ModeIcon tooltip titles consistent with those ones [below](https://github.com/CompuIves/codesandbox-client/pull/239/files#diff-1f317ba5bd25c442a32513566177d0c3R264). Currently we have them as `Editor view`, `Split view` and `Preview view`

Reply:
Ah, I didn't notice those.

I'd like to suggest moving from _this is the name of the thing you're clicking on_ to a more verb-oriented  _this is what will happen when you click_ style of descriptions. I'd like to change the existing Tooltips to reflect my changes, if possible.

Reply:
Let's keep the existing unchanged and change ours to `Show Editor view`, `Show Split view` and `Show Preview view`. What do you say?

Reply:
I like it! Will do.